source/blog.html.haml

```diff
@@ -0,0 +1,32 @@
+---
+title: Blog
+pageable: true
+per_page: 5
+---
+
+%h2.blog_heading
+  Euro Team Outreach Blog Articles
+
+%section.narrow_container
+  - page_articles.each_with_index do |article, i|
+    - pretty_date = article.date.strftime("%b %d, %Y")
+
+    %article.article-preview
+      .article-preview__title
+        %h3
```

Comment:
I think these should be `h2` elements. That's what you're targeting in your css. Right now, the spacing is way off.

https://www.dropbox.com/s/a531gcvapp1vdia/Screenshot%202017-08-17%2019.13.18.png?dl=0